src/controllers/ScheduleController.js

```diff
@@ -469,10 +469,17 @@ router.post('/:id/edit', isAdmin, async (req, res) => {
         const oldGameTime = existingGame.time;
         const oldGameduration = existingGame.duration;
 
-        if (team1 === undefined) {
+
+        // Check if the selected team values are dummy names if true retain the existing dummy teams
+        if (team1 === existingGame.opponents[0].name && existingGame.opponents[0].isDummy) {
+            team1 = existingGame.opponents[0];
+        } else if (team1 === undefined) {
             team1 = existingGame.opponents[0];
         }
-        if (team2 === undefined) {
+
+        if (team2 === existingGame.opponents[1].name && existingGame.opponents[1].isDummy) {
+            team2 = existingGame.opponents[1];
+        } else if (team2 === undefined) {
```

Comment:
This fixes the Dummy overwriting by maintaining the dummi game and give it to the edit page as an option to select

Reply:
sounds good - i will think about this closer, when i do the scheduling logic